packages/web-react/src/components/Button/__tests__/Button.test.tsx

```diff
@@ -37,4 +37,11 @@ describe('Button', () => {
     const element = dom.container.querySelector('button') as HTMLElement;
     expect(element.textContent).toBe('Hello World');
   });
+
+  it('should render Spinner', () => {
```

Comment:
What about having a common test for the loading indicator? E. g. specific test only for `isLoading` prop as we have it for dictionaries...